### ISSUE TITLE
7.x update

### ISSUE THREAD

**Issue by hazelora** [UnknownX7/OOBlugin#3]
plugin doesn't work after dawntrail and dalmud update

**Comment by Sunflowertism** [comment]
Is this one going to get an update by chance

**Comment by Sunflowertism** [comment]
@UnknownX7 

(づ｡◕‿‿◕｡)づ

**Comment by Sunflowertism** [comment]
Any hope for an update? D: @UnknownX7

**Comment by NCC-Lykos** [comment]
....Well shit I forgot what this plugin even does haha

**Comment by Ynelkish** [comment]
I used that plugin that allows to /sit /doze anywhere with some macros

**Comment by topazic** [comment]
Was hoping we'd get an update for this soonish, just adding my voice to the people who do want one :)

**Comment by Sunflowertism** [comment]
> ....Well shit I forgot what this plugin even does haha

> I used that plugin that allows to /sit /doze anywhere with some macros

Yes, it allows you to use your emotes with a lot more freedom and frequency, with macros etc, it's a small thing but adds a lot of quality of life imo.  Still hoping an update will happen.

**Comment by Ynelkish** [comment]
If someone know another way (without using Cheat Engine), you're welcome ^^

**Comment by YsEmei** [comment]
> If someone know another way (without using Cheat Engine), you're welcome ^^

You can with ReAction, also by UnknownX7

![image](https://github.com/user-attachments/assets/57756cd3-cab7-4872-8304-3dcc21bd899c)

**Comment by AthenaTaso** [comment]
I was able to update a fork of this with hopes of proposing a back merge but the code uses memory scanning for a certain pattern to do most of the actions. The APIX update is done but many of the memory addresses have changed. Doemote, freezegame, frz, and capfps all still work but the others are broken.

I don't feel comfortable PRing this without all the actions fixed and i am not particularly interested in trying to find the values myself. If anyone decides they'd like to help find those memory patterns (with cheatengine) then let me know.

**Comment by Sunflowertism** [comment]
Any hope this will ever get an update?
